docs/managing-data-quality/snowplow-event-recovery/running/flink/index.md

```diff
@@ -16,15 +16,31 @@ sbt flink/assembly
 
 #### Running
 
-Using flink CLI:
+Event recovery jobs are usually ran using our [dataflow-runner](https://github.com/snowplow/dataflow-runner) application. An example configuration for running is [available in the repository](https://github.com/snowplow-incubator/snowplow-event-recovery/blob/master/.dataflow-runner).
+
+To run a transient EMR cluster and execute the job, run:
 
 ```bash
-flink run \
-  snowplow-event-recovery-flink-0.2.0.jar \
-  --input s3://bad-rows-location/** \
-  --output recovered-kinesis-topic \  
-  --failedOutput s3://unrecovered-collector-payloads-location/ \
-  --unrecoverableOutput s3://unrecoverable-collector-payloads-location/ \
-  --config $JOB_CONFIG \
-  --resolver $RESOLVER_CONFIG
-```
+dataflow-runner run-transient \
+  --emr-playbook $REPO_DIR/spark-playbook.json.tmpl \
+  --emr-config $REPO_DIR/spark-cluster.json.tmpl \
+  --vars bucket,$BUCKET_INPUT,region,AWS_$REGION,subnet,$AWS_SUBNET,role,$AWS_IAM_ROLE,keypair,$AWS_KEYPAIR,client,$JOB_OWNER,version,$RECOVERY_VERSION,config,$RECOVERY_CONFIG,resolver,$IGLU_RESOLVER,output,$KINESIS_OUTPUT,inputdir,$BUCKET_INPUT_DIRECTORY,interval,$INTERVAL
```

Comment:
```suggestion
  --vars bucket,$BUCKET_INPUT,region,$AWS_REGION,subnet,$AWS_SUBNET,role,$AWS_IAM_ROLE,keypair,$AWS_KEYPAIR,client,$JOB_OWNER,version,$RECOVERY_VERSION,config,$RECOVERY_CONFIG,resolver,$IGLU_RESOLVER,output,$KINESIS_OUTPUT,inputdir,$BUCKET_INPUT_DIRECTORY,interval,$INTERVAL
```

Reply:
Is there any documentation for all these vars? As a user I would have no idea what to put in many of them